src/applications/vaos/reducers/appointments.js

```diff
@@ -38,6 +41,13 @@ const initialState = {
   facilityData: {},
   requestMessages: {},
   systemClinicToFacilityMap: {},
+  expressCare: {
+    fetchWindowsStatus: FETCH_STATUS.notStarted,
```

Comment:
I did a double take when I saw this name in the RequestExpressCare component, because typically we start actions with the word `fetch`, but not Redux properties. What do you think about changing it to something like `windowsStatus`?

Reply:
Good call. Renamed to `windowsStatus`